src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -149,4 +172,76 @@ private ConsortiaConfiguration createConsortiaConfigurationBody(String tenantId)
     configuration.setCentralTenantId(tenantId);
     return configuration;
   }
+
```

Comment:
all shadow user/permissions logic should not be located in the TenantService

Reply:
As option you can create ShadowUserService that will be injected in both: UserTenantService and TenantService

Reply:
The logic for creating ,getting ,adding permissions is already shifted to PermissionService Class. In this class the logic is totally related to the Tenant bussiness.
Same with shadow user operation , it is coimng from userTenantService.

Kindly suggest. thanks

Reply:
Decided to introduce PermissionService to encapsulate common logic including reading permissions list from file path that also will be used for system user story(system user also need to read permissions from file path)
CC @azizbekxm